app/components/provider_interface/completed_offer_summary_component.rb

```diff
@@ -20,10 +20,10 @@ def rows
           value: course_option.site.name_and_address,
           action: 'Change',
           change_path: change_location_path },
-        { key: 'Accredited body',
-          value: course_option.course&.accredited_provider&.name_and_code || course_option.provider.name_and_code,
-          change_path: nil },
       ]
+      return rows unless course_option.course&.accredited_provider&.name_and_code
```

Comment:
Nit, we're not interested in `name_and_code`

```suggestion
      return rows unless course_option.course.accredited_provider.present?
```